tests/compilation_tests/grid_test.py

```diff
@@ -135,6 +135,73 @@ def test_is_on_border(self):
 
         self.assertEqual(len(g.border_nodes), count)
 
+    def test_ActivateRectangle(self):
+        lat_min, lat_max, long_min, long_max = 42.444250, 42.444599, -76.483682, -76.483276
+        g = Grid(lat_min, lat_max, long_min, long_max)
+        
+        my_border = []
+        my_border += list(g.nodes[0][1:-1])
+        my_border += list(g.nodes[38][1:-1])
```

Comment:
since these are hardcoded, let's add a comment why we chose 38 (doesn't have to be long)